src/main/java/io/vertx/ext/sql/ResultSet.java

```diff
@@ -142,11 +143,26 @@ public ResultSet setNext(ResultSet next) {
    * @return  the rows represented as JSON object instances
    */
   public List<JsonObject> getRows() {
+    return getRows(false);
```

Comment:
Also, it might be worth a note in the breaking changes that now rows lookup is case insensitive by default.

Reply:
It's not breaking, by default you get the old behavior, `false` means case sensitive

Reply:
Right, sorry